app/services/create_batch.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
+                            .includes(:appointment_summaries_batches)
+                            .where('appointment_summaries_batches.appointment_summary_id': nil)
+
+    return nil if appointment_summaries.empty?
+
+    Batch.create(appointment_summaries: appointment_summaries)
```

Comment:
The intention is that this class will be consumed by *another* class, something along the lines of `ProcessOutputDocuments` which would then coordinate the CSV generation and SFTP upload. Without the consumer in place the interface to this is a bit theoretical.

My assumption was that this call would return a `Batch` only if there were `AppointmentSummary`s which required processing and which were then batched up accordingly.